internal/artifacts/meta.go

```diff
@@ -22,10 +22,10 @@ func ExecuteGenMeta(source, target, desc string, extensions []string, platform s
 	if err != nil {
 		return errors.Wrap(err, "failed to generate metadata when initializing the location")
 	}
-	return executeGenMetaByLocation(loc, loc, platform, true)
+	return executeGenMetaByLocation(loc, loc, platform, true, false)
 }
 
-func executeGenMetaByLocation(loc *dir.Loc, targetArtifacts dir.ITargetArtifacts, platform string, createMetaInf bool) error {
+func executeGenMetaByLocation(loc *dir.Loc, targetArtifacts dir.ITargetArtifacts, platform string, createMetaInf bool, skipValidation bool) error {
```

Comment:
I think the parameter should be `validatePaths` instead of `skipValidation`. It's more meaningful. (The value should be the opposite if you change the parameter name, of course.)